src/containers/InteractionButton/index.js

```diff
@@ -108,7 +110,11 @@ class InteractionButton extends Component {
 
     if (networkError) {
       return (
-        <Tooltip overlay="You are connected to the wrong chain. You can only interact with Gnosis when you're on the same chain as us.">
+        <Tooltip
+          overlay={`You are connected to the wrong chain. You can only interact using ${
+            ETHEREUM_NETWORK_IDS[targetNetworkId]
```

Comment:
Please display the network in normal casing using `upperFirst(networkName.toLowerCase()`: https://lodash.com/docs/4.17.4#upperFirst

Maybe move to a separate variable for the overlay text